### ISSUE TITLE
[Documentation Update]: the websites footer  color style is not contrast with the header color

### ISSUE THREAD

**Issue by Anu142004** [Trisha-tech/OnlineBookSales#489]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue Description

It is good to keep the header and footer color as same it will contrast each other in which it could help to create a better looking website .

### Suggested Change

change the footer with same color as the header

### Rationale

to make the website more appearance and good looking 

### Urgency

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

**Comment by Suman373** [comment]
> ### Is there an existing issue for this?
> 
>     * [x]  I have searched the existing issues
> 
> 
> ### Issue Description
> 
> It is good to keep the header and footer color as same it will contrast each other in which it could help to create a better looking website .
> ### Suggested Change
> 
> change the footer with same color as the header
> ### Rationale
> 
> to make the website more appearance and good looking
> ### Urgency
> 
> High
> ### Record
> 
>     * [x]  I have read the Contributing Guidelines
> 
>     * [x]  I'm a GSSOC'24 contributor
> 
>     * [x]  I have starred the repository

You may provide some screenshots for better clarity.

**Comment by Anu142004** [comment]
ok

**Comment by Anu142004** [comment]
shall i work on this

**Comment by Suman373** [comment]
> shall i work on this

Yes u can. Am assigning this to you. Good luck!

**Comment by Anu142004** [comment]
thank you

**Comment by Anu142004** [comment]
pl do send the html file

**Comment by Trisha-tech** [comment]
> pl do send the html file

There is only one index.html file which is the starting file of the project. All the code is written in reactjs. Do you know ReactJS? @Anu142004

**Comment by Anu142004** [comment]
no

**Comment by officeneerajsaini** [comment]
> no

Do you need any help ? @Anu142004

**Comment by Anu142004** [comment]
Thank you

**Comment by Suman373** [comment]
> @Suman373, I found an issue with this repo's live web code. The repo's code differs slightly from each other.
> <img alt="Screenshot 2024-10-22 at 8 52 06 PM" width="1440" src="https://private-user-images.githubusercontent.com/118799941/378906082-019a1b0c-bf34-41f7-b749-cae87b76dc3b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk2MTg5MTEsIm5iZiI6MTcyOTYxODYxMSwicGF0aCI6Ii8xMTg3OTk5NDEvMzc4OTA2MDgyLTAxOWExYjBjLWJmMzQtNDFmNy1iNzQ5LWNhZTg3Yjc2ZGMzYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyMlQxNzM2NTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zZjg0ZjVhMGZjYTQwNjg1YmIwODQ3MGI3YWI4YzNiZDAwYWU3OTFkYjFiMjM2NGI3ZWU1NzcyYzNlMjk4YTJmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.bhW4LP1ABY66rzeLdFeFN1eaBn3dr3DA78YfEhcy3kY"> <img alt="Screenshot 2024-10-22 at 8 51 34 PM" width="1440" src="https://private-user-images.githubusercontent.com/118799941/378906059-697018a2-848e-46de-b279-4f3920f0644a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk2MTg5MTEsIm5iZiI6MTcyOTYxODYxMSwicGF0aCI6Ii8xMTg3OTk5NDEvMzc4OTA2MDU5LTY5NzAxOGEyLTg0OGUtNDZkZS1iMjc5LTRmMzkyMGYwNjQ0YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyMlQxNzM2NTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jYWIwYmUyOTg4ODg4YzczYWJjYmM1NzFlNTc5NzM1MTQwZTEyZjI5YmMwMWZmMTM4NmJhNTVhYjYxMjlhODdhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.fKI0NvMdxKEHfY2pNpkEyG5StR-yJTPSv0iHzDNePJk">
> 
> Header and Footer class properties are a mismatch

Hey ! Thanks for notifying us, I'll talk to the PA regarding this.

**Comment by officeneerajsaini** [comment]
@Trisha-tech  please review this issue.

**Comment by Suman373** [comment]
Btw you were not assigned to this yet. @officeneerajsaini

**Comment by officeneerajsaini** [comment]
Yes, I want to work on it. @Suman373 Please assign to me .

**Comment by officeneerajsaini** [comment]
@Suman373 have you asked the maintainer why both websites are different? @Trisha-tech Please Take a Look at this issue.

**Comment by Suman373** [comment]
> @Suman373 have you asked the maintainer why both websites are different? @Trisha-tech Please Take a Look at this issue.

Yes, the deployed version points to a different commit and not the latest one. PA will update it soon.
